docs/UserGuide.md

```diff
@@ -459,6 +463,7 @@ Format: `add -p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_
 
 * Adds a task to a patient at the specified `PATIENT_INDEX`.
```

Comment:
Please have a look at the other feature sections and follow the language style as this may make it look inconsistent.

Reply:
should be using 2nd-person language

Reply:
The summary of what the command does should be placed before `Format:`